libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -438,6 +438,16 @@ bool AP_Arming::battery_checks(bool report)
 
 bool AP_Arming::hardware_safety_check(bool report) 
 {
+    // always check that we a safety change is not occuring
+    // this should never occur, but if we are unable to force the safety this
+    // could lead to moving the safety state in flight
+    if (hal.rcout->is_forcing_safety()) {
+        if (report) {
+            gcs().send_text(MAV_SEVERITY_CRITICAL, "PreArm: Forcing safety");
```

Comment:
I don't like the message, I think it's confusing to the user. I would write something like *Changing safety switch state*